npm-shrinkwrap.json

```diff
@@ -1,11 +1,1823 @@
 {
```

Comment:
I don't think we want to update this file. Just revert it back to what it was. I think this happened because of your nodejs version.